app/lib/actions/katello/content_view_version/incremental_update.rb

```diff
@@ -166,8 +166,9 @@ def generate_description(version, content)
             end
             humanized_lines << ''
           end
-          version.description = humanized_lines.join("\n")
-          version.save!
+          content_view = version.content_view
+          content_view.description = humanized_lines.join("\n")
+          version.save! && content_view.save!
```

Comment:
I don't think this is what we want.  Shouldn't this be saving it to the history object rather than the content view's description ?

A content view has its own description separate from its many versions.